src/components/project/dto/create-project.dto.ts

```diff
@@ -21,10 +21,28 @@ export abstract class CreateProject {
   readonly type: ProjectType;
 
   @IdField({
-    description: 'A country ID',
+    description: 'A primary location ID',
     nullable: true,
   })
-  readonly locationId?: string;
+  readonly primaryLocationId?: string;
+
+  @IdField({
+    description: 'A non primary location ID',
+    nullable: true,
+  })
+  readonly nonPrimaryLocationId?: string;
```

Comment:
Other locations need to be a list. A list of IDs here would be ok on create, but for updating we'd need separate add/remove mutations.

Reply:
I'd also be ok completely skipping this feature for now. It's new and not needed for MVP launch.